cdec/installation-manager-resources/src/main/resources/im-scripts/install-codenvy.sh

```diff
@@ -1524,8 +1561,8 @@ doCheckAvailableResourcesOnNodes() {
                 ;;
         esac
 
-        # check on OS CentOS 7
-        if [[ "${osType}" != "CentOS" || "7.1" > "${osVersion}" ]]; then
+        # check on CentOS or RHEL OS of version >= 7.1
+        if [[ "${osType}" != "CentOS" || "${osVersion}" < "${MIN_OS_VERSION}" ]]; then
```

Comment:
here we don't check RHEL ? (comment is saying checking RHEL)
also as we already have a check lines before
should we have a function for this test to avoid duplicates ?